src/lib/EventPublisher.js

```diff
@@ -95,6 +97,8 @@ class EventPublisher extends EventEmitter {
           responseListener,
           {
             once: true,
+            onceUnsubscribe: true,  // Unsubscribe will be called by Eventpublisher
```

Comment:
Yup, was also my first idea. But this means you you are not allowed (if you use once) to call unsubscribe (even if you got a subscription ID) cause this will throw an error. So the idea was to enable the Automatic Unsubscribe with this flag